### DISCUSSION TITLE
请问这个包用的是v3的包吗

### DISCUSSION THREAD

**Discussion by Malus-risus** [diana7127/mpv.net-DW#15] [Q&A] [answered]
请问这个包用的是v3的包吗

**Comment by Malus-risus** [comment]
v3指的是x86_64_v3
![S30807-09500992_com android chrome](https://github.com/diana7127/mpv.net-DW/assets/69906215/66433480-ba85-43f8-88e2-0d2a9d82e9a9)

**Comment by diana7127** [comment, marked as answer]
v1.6.0用的是v3，v1.5.0不是v3
